bcrs-ui/vue.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  configureWebpack: {
+    devtool: 'source-map'
+  },
+  transpileDependencies: [
+    'vue-plugin-helper-decorator',
+    'vuetify'
+  ]
+}
```

Comment:
Do you need this line here as well, or does this work as is?
`publicPath: process.env.VUE_APP_PATH,`
(this matches the properties in .env)

Reply:
The existing code does work as is.  I debated included this until we knew what our path would be, but on second thought we should include this code even though its a placeholder of sorts so it is not forgotten later. Thank you!

Reply:
Why do you need to prepend a \ for public path (where coops-ui doesn't)?

Reply:
That is a good question, i'm not sure if it was because i'm running it at localhost or because something is happening with the pathing behind the scenes in webpack, but in this instance it was required to set the correct path and find the public config file.

Reply:
OK for now.